doc/release/v2_3_70_2.md

```diff
@@ -22,6 +22,11 @@ Important Changes
 * The `RandScalar` methods are now using C++11 `mersenne_twister_engine` instead
   of a custom engine.
 
+#### `YARP_dev`
```

Comment:
@drdanz it is the wrong changelog file, remember to move it in `v2_3_71.md` before the release!

Reply:
Thanks for catching this!